composer.json

```diff
@@ -17,7 +17,12 @@
         "rebelcode/module-iterator-abstract": "dev-develop",
         "rebelcode/module-locator": "dev-develop",
         "rebelcode/module-config-interface": "dev-develop",
-        "rebelcode/module-config-validator": "dev-develop"
+        "rebelcode/module-config-validator": "dev-develop",
+        "rebelcode/module-factory-abstract": "dev-develop",
+        "dhii/data-key-value-aware-trait": "dev-task/initial-traits",
+        "dhii/callback-aware-trait": "dev-task/initial-traits",
+        "dhii/config-aware-trait": "dev-task/initial-traits",
```

Comment:
Sure. I'll move them in. But do you think they deserve to be in the `RebelCode\Modular` namespace?